WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1413,20 +1443,11 @@ public void updatePostContentNewEditor(boolean isAutoSave, String title, String
             // TODO: Shortcode handling, media handling
         }
 
-        String moreTag = "<!--more-->";
-
         post.setTitle(title);
-        // split up the post content if there's a more tag
-        if (post.isLocalDraft() && content.contains(moreTag)) {
```

Comment:
So with [the move](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/77/commits/cda4601c7777336bfec1e06069be25da201fe837) to the [`wp`](https://codex.wordpress.org/XML-RPC_WordPress_API/Posts) XML-RPC methods for posts (away from [`metaWeblog`](https://codex.wordpress.org/XML-RPC_MetaWeblog_API#metaWeblog.getPost)), we eliminated the need for separating posts into `description` and `more` text, since they're always received and sent as one single param, `post_content`.

The WP.com REST API does [the same thing](https://developer.wordpress.com/docs/api/1.2/post/sites/%24site/posts/new/).